.changeset/curly-gorillas-pull.md

```diff
@@ -0,0 +1,31 @@
+---
+'braid-design-system': minor
+---
+
+---
+updated:
+  - Button
+  - ButtonLink
+---
+
+**Button, ButtonLink:** Default to ghost neutral in non-legacy themes
```

Comment:
Nitpick: the tone and variant ordering flip flops throughout this changeset. Should it be more consistent? E.g always prefer mentioning the `tone` before the `variant`?
```suggestion
**Button, ButtonLink:** Default to ghost neutral in non-legacy themes
```

Reply:
Yeah can do.